src/containers/Dev.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useRouteData } from 'react-static';
+import { Link } from '@reach/router';
+
+const Devops = () => {
```

Comment:
```suggestion
const DevOpsProject = () => {
```
Component names should use `PascalCase` and be unique (or unique enough so as not to confuse them with other components. This component would be easily confused with the devops page component)